cmsdb/processes/ewk.py

```diff
@@ -734,14 +795,18 @@
     label="ZZ",
     xsecs={
         13: Number(15.99, {"scale": (0.037j, 0.026j)}),
+        # 13.6 from GenXSecAnalyzer:
+        13.6: Number(12.84, {
+            "tot": 0.006035,  # xsdb: Number(12.75, {"tot": 0.0649})
+        }),
     },
 )
 
 # based on GenXSecAnalyzer
 # for ZZTo2Q2L_mllmin4p0_TuneCP5_13TeV-amcatnloFXFX-pythia8 (Summer20UL16, NLO)
 # using command ./calculateXSectionAndFilterEfficiency.sh -f datasets.txt -c RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2 -n 5000000  # noqa
-zz_qqll_m4 = zz.add_process(
-    name="zz_qqll_m4",
+zz_zqq_zll_m4toinf = zz.add_process(
+    name="zz_zqq_zll_m4toinf",
```

Comment:
Not really part of the changes from this PR, but since I am currently seeing this, I mentioned these Processes also in #51. Are these `m4toinf` Processes really necessary? I compared the cross sections from these processes with the cross section one gets by multiplying the inclusive XS with the BRs and it looked like the inclusive cross sections were typically much smaller than the `m4toinf` cross section. I'd therefore take another look at these cross sections before using them in an analysis.

Reply:
There are some datasets in this phasespace that we might need in the future.

Reply:
I understand that, but how large is the cross section difference between m0toinf and and m4toinf? Is that significant? I feel like neglecting this difference would be more valid than using cross sections from datasets that are totally off, for example:
```
In [15]: zz.xsecs[13].nominal
Out[15]: 15.99

In [16]: (zz.xsecs[13] * const.br_zz["llll"]).nominal
Out[16]: 0.16303002132924002

In [17]: zz_llll_m4.xsecs[13].nominal
Out[17]: 1.325
```

It might of course also be the case that other decay channels are also included in that dataset as well, but then the process would be named incorrectly.

I checked all the zz processes, and except for `zz_nunuqq`, all cross sections from the `_m4` datasets are larger than the cross section one obtains from multiplying the inclusive XS with the branching ratio.

Reply:
I see. I will validate where these numbers are coming from.

Reply:
I updated the inclusive zz cross section. The previous value referred a paper from 2011 and is more than 50% off the latest calculation I could find: https://link.springer.com/article/10.1007/JHEP03(2019)070#preview

The new value is also far more compatible to the `m4toinf` sub phasespaces.

Reply:
Adjusted in [baa8d77](https://github.com/uhh-cms/cmsdb/pull/46/commits/baa8d77bd8a5489622781805f1bc53d70ec99969).